.stylelintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "@18f/identity-stylelint-config",
+  "rules": {
+    "selector-class-pattern": [
+      "^(tablet:)?[a-z]([a-z0-9-]+)?(__([a-z0-9]+-?)+)?(--([a-z0-9]+-?)+){0,2}$",
```

Comment:
There's a couple style selectors using USWDS's responsive prefixes (`.tablet:`, etc.). Might be worth considering to incorporate this in the main configuration.

Example selector: https://github.com/18F/identity-site/blob/6b25a0b0104703addeed7020307e0867a1f528d0/_sass/pages/_home.scss#L40-L42

Original rule: https://github.com/18F/identity-idp/blob/549bef0aaa0c0f098dce8e44e89548e523ad7631/app/javascript/packages/stylelint-config/index.js#L10-L16